framework/auth/core.py

```diff
@@ -74,7 +74,9 @@ def validate_social(value):
 
 
 def get_current_user_id():
-    return session._get_current_object() and session.data.get('auth_user_id')
```

Comment:
Question/Comments

* When we use `session` from the local proxy, need to understanding why use  `_get_current_object()` and what are the pos and cons. 
> def _get_current_object(self):
> """Return the current object.  This is useful if you want the real
> object behind the proxy at a time for performance reasons or because
> you want to pass the object into a different context.
> """

* In addition, the old code check if `session._get_current_object()` exists. If so it returns session.data.get('auth_user_id'); otherwise, it returns `session._get_current_object()`. Not sure why we needed it and whether we still need it.